templates/default/http_realip.conf.erb

```diff
@@ -0,0 +1,15 @@
+# This config file enables http_realip module
+# some `set_real_ip_from` can be set here
+# other can be contained in separate files i.e. `cloudflare-ips`
+#
+# set_real_ip_from x.x.x.x/xx;
+# set_real_ip_from 0000:0000::/xx;
+
+<% if @params['real_ip_from'] %>
```

Comment:
If the template declares:
```
variables({
    :real_ip_from => node['nginx']['real_ip_from'],
    :real_ip_header => node['nginx']['real_ip_header'],
    :real_ip_recursive => node['nginx']['real_ip_recursive']
  })
```, the variables in this template would be accessed like: `@real_ip_from`, `@real_ip_header` and `@real_ip_recursive`.

If you want to use @params, you would need to pass the variables in as:
```
  variables(params: {
    real_ip_from: node['nginx']['real_ip_from'],
    real_ip_header: node['nginx']['real_ip_header'],
    real_ip_recursive: node['nginx']['real_ip_recursive']
  })
```